public/app/features/alerting/unified/components/receivers/form/CloudReceiverForm.tsx

```diff
@@ -53,7 +52,7 @@ export const CloudReceiverForm: FC<Props> = ({ existing, alertManagerSourceName,
         oldConfig: config,
         alertManagerSourceName,
         successMessage: existing ? 'Contact point updated.' : 'Contact point created.',
-        redirectPath: makeAMLink('/alerting/notifications', alertManagerSourceName),
```

Comment:
Question: is there a particular reason why we no longer want to redirect to the current alertmanager datasource here?

Reply:
We want :) we use `makeAMLink` in the `updateAlertManagerConfigAction` which builds `makeAMLink` based on `redirectPath` and `alertManagerSourceName`. This `makeAMLink` call results in a duplicated `alertmanager` query parameter (Beacause we call makeAMLink twice on the same URL). I aligned this to how it works in `GrafanaReceiveForm`